src/test/java/com/appdirect/sdk/credentials/SourceBackedCredentialsTest.java

```diff
@@ -0,0 +1,50 @@
+package com.appdirect.sdk.credentials;
+
+import com.appdirect.sdk.appmarket.Credentials;
+
+import java.util.function.Function;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SourceBackedCredentialsTest {
+
+	private SourceBackedCredentials findCredentials;
```

Comment:
Use a more descriptive name
```suggestion
	private SourceBackedCredentialsSupplier sourceBackedCredentialsSupplier;
```